### PR TITLE
Add simple board and python class

### LINKED ISSUE #8: Integrate simple board w websocket
Acceptance Criteria:

- Add Chess board Initialization to app.py
- A hard coded move message from UI to Chess Board 
- A Board Update Message from Engine to UI

### PR DESCRIPTION
**Pull request by GarettOS**
closes #8

### CODE REVIEW

**Comment by GarettOS** [comment]
> [Feb 28, 2024](#event-11960680749)
done

**Comment by GarettOS** [comment]
> I don't see this integrated with the websocket

done

File: engine/__pycache__/board.cpython-38.pyc
Comment (walshb421) [inline engine/__pycache__/board.cpython-38.pyc:1]: You should delete this directory from your commit:
```bash
git rm -rf ./engine/__pycache__/
git commit -m "Removed Python Cache files"
```

**Review by walshb421 — CHANGES_REQUESTED** [review]
This looks awesome! I was able to move a pawn, got it to crash somehow but that is to be expected at this point! Excited to get this merged so that I can get working on adapting my board to it. If you delete those files and push, it I will be satisfied!

**Review by walshb421 — APPROVED** [review]

**Pull request merged**